app/patients/reports/controller.js

```diff
@@ -568,13 +568,17 @@ export default AbstractReportController.extend(PatientDiagnosis, PatientVisits,
           admissionDate: visit.get('startDate'),
           dischargeDate: visit.get('endDate')
         };
-        var sexGrouping = patientBySex[visit.get('patient.sex')];
+        var sex = visit.get('patient.sex');
+        if (!sex) {
+          sex = 'Sex Not Entered';
```

Comment:
@akong Can you localize the text here?  You can put the label under patient.labels.sexNotEntered in the localization file at app/locales/en/translations.js and then you can pull it in to the controller by calling
```javascript
this.get('i18n').t('patients.labels.sexNotEntered'),
```